getting-started/vs-integration/upgrade-wizard.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Upgrading Projects
+page_title: Upgrade Project - Visual Studio Integration
+description: Learn how to upgrade a Telerik UI for Blazor project with our Visual Studio Extension.
+slug: getting-started-vs-integration-upgrade-project
+position: 6
+---
+
+# Upgrading Projects
+
+This article demonstrates how to use the [Telerik Visual Studio Extension]({% slug getting-started-vs-integration-overview %}) to upgrade a project that is pre-configured for the Progress&reg; Telerik&reg; UI for Blazor components.
+
+## The Basics
+
+The Upgrade Project Wizard upgrades the version of an existing Telerik UI for Blazor applications. To open the wizard, right Click on the **Project in the Solution Explorer -> Telerik UI for Blazor -> Upgrade Wizard**.
+
+![Open Upgrade Wizard](images/upgrade-wizard-open.png)
+
+## Files for Upgrade
+
+The **Information Page** of the Upgrade Project Wizard describes the files and assemblies that are going to be upgraded.
+
+![Upgrade Wizard Information Page](images/upgrade-wizard-information.png)
+
+* The Telerik Blazor Upgrade Wizard will do the following: 
```

Comment:
```suggestion
The Telerik Blazor Upgrade Wizard will do the following: 
```
No need for a bullet here, the sentence is not part of a bullet list. You may render plain text or caption.